ui/src/components/dashboard/audio_list_item.test.coffee

```diff
@@ -54,6 +54,25 @@ describe 'audio-list-item', ->
   it 'should not have an invisible icon', ->
     expect(@wrapper.find('.fa.fa-eye-slash')).to.have.length 0
 
+  describe 'list item accent color', ->
+    extension_check = (ext, color) =>
+      it 'should be '+color+' on '+ext, ->
+        new_fixture = Object.assign {}, audios_fixture.chicken, url: 'chicken.' + ext
+        propsData = 
+          audio: new_fixture
+        @wrapper = mount audio_list_item, globals: ($store: @store), propsData:
+          audio: new_fixture
```

Comment:
duplicate... propsData on the line above is unused

Reply:
Whoops, my bad! This was from one of my failed attempts, forgot to delete 🤦‍♂️